tests/Composer/InfoCommandTest.php

```diff
@@ -33,7 +33,7 @@ class InfoCommandTest extends PHPUnit_Framework_TestCase
 
     protected function setUp()
     {
-        $this->application = $this->getMock('Magento\Composer\MagentoComposerApplication', [], [], '', false, false);
+        $this->application = $this->createMock(Magento\Composer\MagentoComposerApplication::class);
```

Comment:
Why leading slash is removed everywhere?

Reply:
Added slashes.